task_12/src/js/time.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+document.addEventListener('DOMContentLoaded', function () {
+    var clock = document.getElementsByClassName("time")[0];
```

Comment:
Probably, you are not sure that clock always be first element with class. Prefer `getElementById` or `querySelector`